### ISSUE TITLE
Some link in xlsx file is not clickable after convert to PDF

### ISSUE THREAD

**Issue by jlemangarin** [ONLYOFFICE/DocumentServer#1101]
Issue following discussions in https://github.com/ONLYOFFICE/DocumentServer/issues/189

**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
Links not clickable in pdf produced from Conversion API : https://api.onlyoffice.com/editors/conversionapi

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Export a PDF using the Conversion API

**What is the expected behavior?**
Links should be clickable as implemented in https://github.com/ONLYOFFICE/DocumentServer/issues/189

**Did this work in previous versions of DocumentServer?**
I think no

**DocumentServer version:**
6.1.0

**Operating System:**
CentOS

**Browser version:**
All

**Comment by ShockwaveNN** [comment]
@jlemangarin Could you attach some sample file you want to convert to pdf and exact parameters you're using in conversionapi

**Comment by avanhoveceovision** [comment]
Hello, 
We used this parameters to convert docx into pdf file: 

```
  $jsonData = array(
    'filetype' =>'docx',
    'key' => xxxxxxx,
    "outputtype" => "pdf",
    "title" => 'testtttt',
    "url" => xxxxx,
  );
```
Here is an exemple of a doc to convert with onlyoffice converter API.


[Testttttt_v1.0.docx](https://github.com/ONLYOFFICE/DocumentServer/files/5696161/Testttttt_v1.0.docx)

**Comment by ShockwaveNN** [comment]
@avanhoveceovision I opened this docx file in ONLYOFFICE DesktopEditors and LibreOffice - there is no visible links in this docx file even before convert to pdf

**Comment by avanhoveceovision** [comment]
hi,

Sorry i did a mistake,

Here is my file :

[Document 1.docx](https://github.com/ONLYOFFICE/DocumentServer/files/5696731/Document.1.docx)

**Comment by ShockwaveNN** [comment]
I converted this file and opened in chrome and link is working correct

Please note that if this file is opened in ONLYOFFICE DocumentServer or ONLYOFFICE DesktopEditors there is a bug: https://github.com/ONLYOFFICE/DocumentServer/issues/1044

Previous bug https://github.com/ONLYOFFICE/DocumentServer/issues/189 was for situation for which any application (including chrome pdf viewer) will not click on links

**Comment by avanhoveceovision** [comment]
@ShockwaveNN Can you try to convert this file and check the link is not clickable ? 

[Feuille de calcul 1_v1.0.xlsx](https://github.com/ONLYOFFICE/DocumentServer/files/5696864/Feuille.de.calcul.1_v1.0.xlsx)

Thank you !

**Comment by ShockwaveNN** [comment]
@avanhoveceovision For your second file first link is ok, second link is not

But this situation happens in DocumentServer itself, it's not some specific bug for convert service, I'll try to find if this is old one

**Comment by avanhoveceovision** [comment]
@ShockwaveNN Thank you !

**Comment by ShockwaveNN** [comment]
I can confirm this is a new bug for us, issue 48031 in our private issue tracker
We've make a note, that this is issue from `ceo-vision.com` and hope our dev team will be take a look at this in higher priority than usual, but cannot guarantee it
Thank you

**Comment by avanhoveceovision** [comment]
Hello @ShockwaveNN!
Thank you !

Greatings from CEO-Vision

**Comment by avanhove** [comment]
Hello, 

Any ETA for this issue ?

Kind regards

**Comment by ShockwaveNN** [comment]
@avanhove Sorry, developers didn't promise me any time frame for this fix

**Comment by cpot** [comment]
@ShockwaveNN Important fix for us. this will allow us to replace our libreoffice preview engine by onlyoffice for spreadsheet

**Comment by ShockwaveNN** [comment]
@cpot Got it, I'll try to get attention of our dev team

**Comment by cpot** [comment]
any ETA for this issue (onlyoffice paying customers are asking us, again, when it will be fixed)
thank you !

**Comment by ShockwaveNN** [comment]
@cpot 

I'm sorry, but as usual, we (QA Team) have little impact on priorities

It's always more productive to contact support@onlyoffice.com / sales@onlyoffice.com if you're paying a customer

So right now I have no knowledge of ETA of this fix, but if you contact support - they may force solution of this problem

**Comment by ShockwaveNN** [comment]
Hi, this issue is fixed via https://github.com/ONLYOFFICE/sdkjs/commit/361816aef71772eed71695a1284660cf2d187fdf

And will be available as v7.1 release I think within a month

You want to check it out - you can try 
```
docker run -itd -p 80:80 onlyoffice/4testing-documentserver-ee:7.1.0.173
```
But please do not use this version in production, it's not 100% production ready
